include/CppUTest/UtestMacros.h

```diff
@@ -249,8 +249,10 @@
 #define BITS_EQUAL_TEXT(expected, actual, mask, text)\
   BITS_LOCATION(expected, actual, mask, text, __FILE__, __LINE__)
 
+#define ROBUST_SIZEOF(type) (2*sizeof(type)/sizeof(short)) /* For the Cl2000, sizeof(char) == sizeof(short) == 1 */
```

Comment:
I'd actually prefer to have the test case under #if BYTE_IS_16_BITS or so.

I don't like the sizeof macro and don't think we need to shrink the sizeof a byte when the platform clearly has 16 bit bytes...